spacecmd/spacecmd.changes

```diff
@@ -1,3 +1,4 @@
+- Add vendor_advisory to errata.getDetails (bsc#1205207)
```

Comment:
`spacecmd` users might not know what `errata.getDetails` means. I'd use the `spacecmd` command(s?) in the changelog that now show this information.

Reply:
Thanks. I made some adjustments and hope it's clearer now